content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -405,6 +410,39 @@ instances:
 ### Serverless environment
 
 For more information about AWS Lambda functions, see [how to connect your Lambda telemetry using tags][15].
+
+### OpenTelemetry environment
+
+In OpenTelemetry environments, you can use the [OpenTelemetry Collector][16] to collect and batch telemetry data (like metrics, traces, and logs) and the [OpenTelemetry Exporter][17] to send the data to Datadog (without the Datadog Agent).
+
+To setup unified service tagging in an OpenTelemetry environment:
+
+1. Set the following attributes in your OpenTelemetry configuration to ensure proper tagging:
+
+   ```yaml
+   service.name: "<SERVICE>"
+   deployment.environment: "<ENV>"
+   service.version: "<VERSION>"
+   ```
+   
+1. To [correlate OpenTelemetry language SDK logs and traces][18], translate the OpenTelemetry `TraceId` and `SpanId` properties into the Datadog format, and ensure your logs are sent as JSON files. Your language-level logs must become Datadog attributes for the trace-log correlation to work.
+
+   The following table outlines the key differences between OpenTelemetry and Datadog schemas for tracing and unified service tagging:
+
+   | Domain | OpenTelemetry | Datadog |
+   |---|---|---|
+   | Tracing | `TraceId` and `SpanId` are stored in OpenTelemetry as 32-hex character and 16-hex character. | Stored in Datadog as 64-bit unsigned int. |
```

Comment:
```suggestion
   | Tracing | `TraceId` and `SpanId` are stored in OpenTelemetry as strings of 32 hex characters and 16 hex characters, respectively. | Stored in Datadog as 64-bit unsigned int. |
```
I'm not 100% sure this was the intended meaning, but the way it was phrased felt odd, since an individual character would just be described as hexadecimal, not 32-hex or 16-hex.